spec/acceptance/onecluster_spec.rb

```diff
@@ -2,23 +2,61 @@
 
 describe 'onecluster type' do
   before :all do
-    skip
     pp = <<-EOS
     class { 'one':
       oned => true,
     }
-    ->
-    onehost { ['host01', 'host02']:
-      ensure  => present, # FIXME: ensurable should default to :present...
+    onehost { 'host01':
+      im_mad => 'kvm',
+      vm_mad => 'kvm',
+      vn_mad => 'dummy',
+      status => 'disabled',
+    }
+
+    onehost { 'host02':
+      im_mad => 'kvm',
+      vm_mad => 'kvm',
+      vn_mad => 'dummy',
+      status => 'disabled',
+    }
+
+    onevnet { 'vnet1':
+      ensure          => present,
+      bridge          => 'basebr0',
+      phydev          => 'br0',
+      dnsservers      => ['8.8.8.8', '4.4.4.4'],
+      gateway         => '10.0.2.1',
+      vlanid          => '1550',
+      netmask         => '255.255.0.0',
+      network_address => '10.0.2.0',
     }
-    ->
-    onevnet { ['Blue LAN', 'Red LAN']:
-      type       => 'fixed',
-      bridge     => 'vbr1',
+
+    onevnet { 'vnet2':
+      ensure          => present,
+      bridge          => 'basebr0',
+      phydev          => 'br0',
+      dnsservers      => ['8.8.8.8', '4.4.4.4'],
+      gateway         => '10.0.2.1',
+      vlanid          => '1550',
+      netmask         => '255.255.0.0',
+      network_address => '10.0.2.0',
     }
+
+    #onedatastore { 'system':
+    #   ensure => present,
+    #}
+
+    #onedatastore { 'default':
+    #  ensure => present,
+    #}
+
+    #onedatastore { 'files':
+    #  ensure => present,
+    #}
+
+
     EOS
     apply_manifest(pp, :catch_failures => true)
-    apply_manifest(pp, :catch_changes => true)
```

Comment:
its still WIP - trying to figure out how to create hosts without making them end up in failed state.